src/index.ts

```diff
@@ -244,16 +250,16 @@ export function offlineInit(config: IClientConfigSync): IEppoClient {
     const memoryOnlyConfigurationStore = configurationStorageFactory({
       forceMemoryOnly: true,
     });
-    memoryOnlyConfigurationStore.setEntries(config.flagsConfiguration);
-    EppoJSClient.instance.setConfigurationStore(memoryOnlyConfigurationStore);
+    memoryOnlyConfigurationStore.setEntries(config.flagsConfiguration).catch();
```

Comment:
`.catch()` added to be more explicit that we are not `awaiting` / ignoring errors.

Reply:
you can `.catch(console.warn)` to fix the lint warning